.github/workflows/build.yml

```diff
@@ -66,11 +66,22 @@ jobs:
         run: echo -e "${GITHUB_REF_NAME}" > cmd/thyra-server/version.txt
       - name: Build binary for ${{ matrix.target }} on ${{ matrix.arch}}
         run: GOOS=${{ matrix.target }} GOARCH=${{ matrix.arch }} CGO_ENABLED=1 CC=${{ matrix.cc }} CXX=${{ matrix.cxx }} go build -o thyra-server_${{ matrix.target }}_${{ matrix.arch }} ./cmd/thyra-server/
+      - name: Build installer binary
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
suggestion:
`if: ${{ matrix.target == 'windows' }}`
could help to understand easier !